readium/navigator/src/main/java/org/readium/r2/navigator/epub/EpubNavigatorFragment.kt

```diff
@@ -655,11 +648,14 @@ class EpubNavigatorFragment private constructor(
         get() = if (::resourcePager.isInitialized) resourcePager.adapter as? R2PagerAdapter
             else null
 
-    private val currentFragment: R2EpubPageFragment? get() =
```

Comment:
This property name was really ambiguous and caused the problem in the first place. It was `null` for an FXL resource.